src/js/jsx/search/SearchBar.jsx

```diff
@@ -126,7 +125,7 @@ define(function (require, exports, module) {
             var idArray = id ? id.split("-") : [],
                 filterValues = _.drop(idArray),
                 updatedFilter = id ? _.uniq(this.state.filter.concat(filterValues)) : [],
-                filterIcon = svgUtil.getSVGClassesFromFilter(updatedFilter);
+                filterIcon = id ? this.getFlux().store("search").getSVGClass(updatedFilter) : null;
```

Comment:
Hot tip: if you've got some expression `foo` which may be falsey, then you can just do `foo && bar` instead of `foo ? bar : null`.